### PR TITLE
Fix author scrape

### PR DESCRIPTION
**Pull request by JGSweets**
This PR:
 * adds reqs
 * fixes author xml
 * spacing
 * updates gitignore

### CODE REVIEW

File: parse_patent.py [hunk 1]
```diff
@@ -102,13 +103,13 @@ def parse_uspto_file(bs, logging=False):
         print("USPTO Publication Number:", publication_num)
         print("USPTO Publication Date:", publication_date)
         print("USPTO Application Type:", application_type)
-            
+
         count = 1
-        for classification in section-class_subclass_groups:
```
Comment (JGSweets) [inline parse_patent.py:107]: fix here

File: parse_patent.py [hunk 2]
```diff
@@ -42,22 +43,22 @@ def parse_uspto_file(bs, logging=False):
...
             section_classes[section+el.find('class').text] = True
             section_class_subclasses[classification] = True
             section_class_subclass_groups[classification+" "+group] = True
-            
+
     authors = []
-    for parties in bs.find_all('parties'):
-        for applicants in parties.find_all('applicants'):
+    for parties in bs.find_all(re.compile('^.*parties')):
+        for applicants in parties.find_all(re.compile('inventors')):
```
Comment (JGSweets) [inline parse_patent.py:61]: author fix

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+bs4
```
Comment (JGSweets) [inline requirements.txt:1]: added

File: utils/db_interface.py [hunk 4]
```diff
@@ -2,25 +2,26 @@
 import csv
```
Comment (JGSweets) [inline utils/db_interface.py:2]: vscode fixed all the spacing

File: .gitignore [hunk 5]
```diff
@@ -2,4 +2,6 @@
 *.xml
 config/**
 *.tsv
-**/__pycache__/**
\ No newline at end of file
+**/__pycache__/**
+.venv/
+venv/
```
Comment (JGSweets) [inline .gitignore:7]: added these two

File: parse_patent.py (hunk 2 above)
Comment (lettergram) [reply parse_patent.py:61]: We probably want to know the entities as well

For instance, I'd like to know all patents from organization X

**Review by lettergram — APPROVED** [review]

**Pull request merged**